src/fstests/fstests.jinja2

```diff
@@ -13,7 +13,7 @@ class Job(BaseJob):
     def _kernel_config(self, kdir):
         try:
             os.chdir(kdir)
-            result = subprocess.run("kvm-xfstests install-kconfig", check=True, shell=True)
+            result = subprocess.run(["kvm-xfstests", "install-kconfig"], check=True, shell=True)
```

Comment:
I suggested to do this as a _follow-up_ which means not in this commit or this PR as it's not related to running the tests...  Adding other random changes makes the code harder to maintain and reviews harder to do.